### PR TITLE
fix customer token issue in api (issue#27098)

### PR DESCRIPTION
**Pull request by magecp**
### Description (*)

Pass website id in the post to resolve this issue.

### Fixed Issues (if relevant)

1. magento/magento2#27098: Magento CE 2.3.X the swagger interface api named integrationCustomerTokenServiceV1 http scheme valid customer acess details entered and execute still server response is 401

### Manual testing scenarios (*)

1. Open the default Magento Swagger HTTP schemes or use this same API in postman.
2. Open integrationCustomerTokenServiceV1 interface to generate a token for customers.
3. Enter valid customer's details and check the response.

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by lbajsarowicz — CHANGES_REQUESTED** [review]
Please cover change with API functional test.

File: app/code/Magento/Customer/Api/AccountManagementInterface.php [hunk 1]
```diff
@@ -105,10 +105,11 @@ public function activateById($customerId, $confirmationKey);
      *
      * @param string $email
      * @param string $password
+     * @param int $websiteId
      * @return \Magento\Customer\Api\Data\CustomerInterface
      * @throws \Magento\Framework\Exception\LocalizedException
      */
-    public function authenticate($email, $password);
+    public function authenticate($email, $password, $websiteId);
```
Comment (lbajsarowicz) [inline app/code/Magento/Customer/Api/AccountManagementInterface.php:112]: As it's `@api` you cannot add the extra required argument.

**Review by lbajsarowicz — CHANGES_REQUESTED** [review]
You cannot add required method argument to `@api`

**Comment by engcom-Charlie** [comment]
Hi @magecp, I'm closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for your collaboration.

**Pull request closed without merging**